Makefile

```diff
@@ -235,19 +227,19 @@ CLEAN_TARGETS += clean-vendor
 clean-vendor:
 	-rm -rf $(VENDOR_DIR)
 
-CLEAN_TARGETS += clean-glide-cache
-.PHONY: clean-glide-cache
-## Removes the ./glide directory.
-clean-glide-cache:
-	-rm -rf ./.glide
-
-$(VENDOR_DIR): glide.lock glide.yaml
-	$(GLIDE_BIN) --quiet install --strip-vendor
-	touch $(VENDOR_DIR)
-
 .PHONY: deps
 ## Download build dependencies.
-deps: $(VENDOR_DIR)
+deps: $(DEP_BIN) $(VENDOR_DIR)
```

Comment:
This effectively removes the necessity to install glide in the Dockerfile: https://github.com/fabric8-services/fabric8-wit/blob/e5d6ca0d52cd26095081a2271cdc9cda020b5af9/Dockerfile.builder#L30 . Please change that file as well and remove glide from there.

Reply:
yes, I need to fix it in here, too